platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratableTest.java

```diff
@@ -80,9 +80,9 @@
  * <p>// This is the system imported into: ./ddf ./ddf/etc ./ddf/Version.txt
  *
  * <p>// The backup from the imported system will look similar to this:
- * ./exported-1.0-20170816T142400.zip
+ * ./exported-1.0-20170816T142400.enc
```

Comment:
❓ `.enc` stands for?

Reply:
encrypted

Reply:
🤔  Maybe we should have an extension for DDF archive, or DDF export. Indicating the file is encrypted, is just asking for it.

Reply:
Any ideas what we should call it then @paouelle? `.dar` (ddf archive)?

Reply:
That would work with me. @lessarderic What do you think?

Reply:
❓ Should these comments be updated to `.dar`?